.github/workflows/docker-publish-opr-node-images.yaml

```diff
@@ -6,6 +6,15 @@ on:
       commit_sha:
         description: 'Specific Commit SHA (Required)'
         required: true
+      version:
+        description: 'Version (Required)'
+        required: true
+      gitCommit:
+        description: 'GitCommit (Required)'
+        required: true
+      gitDate:
```

Comment:
We can compute gitDate from commit. Something like `git show -s --format=%ci <commit>` works